src/docker_compose.rs

```diff
@@ -116,6 +118,17 @@ async fn generate_docker_compose_content(
     ];
 
     let service_images = get_services_images(dev_conf).await;
+    let mut app_ports = vec![];
+    if exposed_services.contains(&Service::App) {
+        app_ports.push(format!("{}:{}", OF_WATCHDOG_PORT, OF_WATCHDOG_PORT));
+    }
+    if debug {
+        if let Some(ports) = dev_conf?.dofigen?.ports {
```

Comment:
This returns none is there is a none in the chain.
It's not the wanted behaviour